orderbook/src/account_balances.rs

```diff
@@ -35,7 +35,7 @@ pub trait BalanceFetching: Send + Sync {
 
 pub struct Web3BalanceFetcher {
     web3: Web3,
-    allowance_manager: H160,
+    vault_relayer: H160,
```

Comment:
Do we only ever use the vault relayer now? Does this mean we don't need the allowance manager anymore? I wasn't aware of this...

Reply:
The allowance manager contract no longer exists in the updated contracts.

Reply:
So I guess there was no point in spending so much time making it upgradable...

Reply:
The allowance manager was never upgradable... I think you may be thinking of authorization contract (i.e. `GPv2AllowListAuthentication`) - which is still there and is still upgradable.